fingertip/plugins/backend/qemu.py

```diff
@@ -129,7 +136,7 @@ def disrupt():
                              stdout=logging.INFO, stderr=logging.ERROR)
     disks = {
         'image.qcow2': disk_size,
-        'efivars.qcow2': '64M'
+        'efivars.qcow2': '4M'
```

Comment:
```
os.alpine.install_in_qemu: qemu-system-aarch64: cfi.pflash01 device '/machine/virt.flash1' requires 67108864 bytes, pflash1 block backend provides 4194304 bytes
```

Works with the original `'64M'` value.

Reply:
OK, let's try different sizes for different arches, as it otherwise it failed for x86_64 OVMF with something like "combined size cannot exceed 64M"